libvast/vast/detail/signal_handlers.hpp

```diff
@@ -0,0 +1,18 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+/// A printing signal handler meant for SIGSEGV and SIGABRT. Prints a backtrace
+/// if support for that is enabled at compile time.
+extern "C" void fatal_handler(int sig);
```

Comment:
Does this need to be marked `extern "C"`? If not, please move to the namespace `vast::detail`.

Reply:
See https://en.cppreference.com/w/cpp/utility/program/signal#Notes.